src/routes/embedded/auth/auth-email-verify/auth-email-verify.view.tsx

```diff
@@ -9,24 +9,29 @@ import { OnboardingCard } from "~components/embed/ui/molecules/card/onboarding-c
 import { Flex } from "~components/common/Flex";
 import { getFriendlyAuthErrorMessage } from "~utils/authentication/authentication.utils";
 import { StorageKeys } from "~utils/storage/storage.constants";
+import { CodeInput, type CodeInputHandle } from "~components/embed/ui/atoms/code-input/CodeInput";
+import { useCooldownCallback } from "~utils/react/useCooldownCallback";
 import {
-  CodeInput,
+  checkNeedsNewOtp,
+  clearOtpAvailable,
   OPT_COOLDOWN_DURATION_SEC,
   OTP_LENGTH,
-  type CodeInputHandle,
-} from "~components/embed/ui/atoms/code-input/CodeInput";
-import { useCooldownCallback } from "~utils/react/useCooldownCallback";
+  setOtpAvailable,
+} from "~utils/otp/otp.utils";
+import { useAsyncEffect } from "~utils/react/useAsyncEffect";
 
 export function AuthEmailVerifyEmbeddedView() {
```

Comment:
This view is only here to handle a very unlikely error when signing in with password to a non-verified account.